apache_exporter.go

```diff
@@ -16,14 +16,14 @@ import (
 	"time"
 
 	"github.com/Lusitaniae/apache_exporter/collector"
+	kingpin "github.com/alecthomas/kingpin/v2"
```

Comment:
```suggestion
	kingpin "gopkg.in/alecthomas/kingpin.v2"
```
I think switching back to gopkg.in will solve the lint error.